layouts/shortcodes/thirdparty-content.html

```diff
@@ -0,0 +1,5 @@
+{{ $color := .Get "color" | default "primary" }}
+<blockquote class="alert alert-{{ $color }}" role="alert">
+  <h4 class="alert-heading">{{ T "caution" | safeHTML }}</h4>
```

Comment:
Hi @celestehorgan .
Thanks for the updates. Yay!
Do you want to add another variable for the string, "Third party content" instead of "Caution"?
Something in `i18n/en.toml` such as `thirdparty_title`?

Reply:
From a technical writing perspective: all callouts should fall under either being a note, warn, or caution. I think a separate title would add complication without really adding meaning :)

Reply:
I just looked at one of the rendered changes:
https://deploy-preview-23107--kubernetes-io-master-staging.netlify.app/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model

With the latest shortcode changes, the third party callout looks like the theme's alert shortcode.
Should the third party content callout look like the note and warning callouts?

Reply:
@kbhawkey from an information perspective – they're all meta-information _about the content_ – i don't see why it should look different.

Reply:
@celestehorgan , I agree that all callout related shortcodes should look the same.
Though, I don't see the third party content in the category of "caution".
I'd rather provide a specific title for the callout (same for feature state, third party, deprecated ...).
/lgtm